src/main/java/org/folio/config/ApplicationConfig.java

```diff
@@ -105,6 +109,36 @@
 @ComponentScan({ "org.folio" })
 public class ApplicationConfig {
 
+  private static final Logger LOGGER = LogManager.getLogger();
+
+  @Value("${KAFKA_HOST:kafka}")
+  private String kafkaHost;
+  @Value("${KAFKA_PORT:9092}")
+  private String kafkaPort;
+  @Value("${OKAPI_URL:http://okapi:9130}")
+  private String okapiUrl;
+  @Value("${REPLICATION_FACTOR:1}")
+  private int replicationFactor;
+  @Value("${MAX_REQUEST_SIZE:4000000}")
+  private int maxRequestSize;
+  @Value("${ENV:folio}")
+  private String envId;
+
+  @Bean(name = "newKafkaConfig")
+  public KafkaConfig kafkaConfigBean() {
+    KafkaConfig kafkaConfig = KafkaConfig.builder()
+      .envId(envId)
+      .kafkaHost(kafkaHost)
+      .kafkaPort(kafkaPort)
+      .okapiUrl(okapiUrl)
+      .replicationFactor(replicationFactor)
+      .maxRequestSize(maxRequestSize)
+      .build();
+
+    LOGGER.info("kafkaConfig: " + kafkaConfig);
```

Comment:
Could you replace the line to built-in formatting like as:  LOGGER.info("kafkaConfig: {}", kafkaConfig); ?

Reply:
done